test/checks/readinessProb/success.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    test: readiness
+  name: readiness-exec
+spec:
+  containers:
+  - name: readiness
+    image: k8s.gcr.io/busybox
+    args:
+    - /bin/sh
+    - -c
+    - touch /tmp/healthy; sleep 30; rm -rf /tmp/healthy; sleep 600
+    readinessProbe:
+      exec:
+        command:
+        - cat
+        - /tmp/healthy
+      initialDelaySeconds: 5
+      periodSeconds: 5
```

Comment:
I think this got left in by mistake 😄 

I can't figure out why the tests are passing for this one (since the check name is `readinessProb`) - it should fail with `check not found`

Reply:
Figured it out! We were only running the `./pkg` tests

https://github.com/FairwindsOps/polaris/pull/532